### PR TITLE
Adjust component

### PR DESCRIPTION
**Pull request by pacospace**
## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Split methods for testings
Add env variables
Adjust main files

### CODE REVIEW

**Comment by pacospace** [comment]
recheck

File: app.sh [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 if [ "$SUBCOMMAND" = "producer" ]
 then
-    exec faust --debug --loglevel debug -A advise_reporter main
-elif [ "$SUBCOMMAND" = "consumer" ]
+    if [ "$DEBUG_LEVEL" -eq 1]
+    then
+        exec faust --debug --loglevel debug -A producer main
+    else
+        exec faust -A producer main
+elif [ "$SUBCOMMAND" = "consumer" ] && [ "$DEBUG_LEVEL" -eq 1]
```
Comment (KPostOffice) [inline app.sh]: no && here

File: app.sh [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
 then
-    exec faust --debug --loglevel debug -A advise_reporter_consumer worker
+    if [ "$DEBUG_LEVEL" -eq 1]
+    then
+        exec faust --debug --loglevel debug -A consumer worker
+    else
+       xec faust -A consumer worker
```
Comment (KPostOffice) [inline app.sh]: Missing e in `exec`

File: consumer.py [hunk 3]
```diff
@@ -18,32 +18,28 @@
...
 import logging
+import os
 
 from thoth.common import init_logging
 from thoth.messaging import MessageBase, AdviseJustificationMessage
-from prometheus_client import start_http_server, Gauge
+from advise_reporter import send_metrics_to_pushgateway
 
-init_logging()
+init_logging({'thoth.advise_reporter': 'INFO'})
```
Comment (KPostOffice) [inline consumer.py]: What does this do?

Comment (pacospace) [reply consumer.py]: https://github.com/goern/thoth-common/blob/8fb41cbf160e7cf5ca328502a62ed417ccb9ad9b/thoth/common/logging.py#L195

Comment (KPostOffice) [reply consumer.py]: Why is it set to `INFO`?

File: advise_reporter.py [hunk 4]
```diff
@@ -15,28 +15,34 @@
...
+    registry=prometheus_registry,
+)
 
-@app.command()
-async def main():
-    """Run advise-reporter."""
-    advise_justification = AdviseJustificationMessage()
+_THOTH_METRICS_PUSHGATEWAY_URL = (
+    os.getenv("PROMETHEUS_PUSHGATEWAY_URL") or "pushgateway-dh-prod-monitoring.cloud.datahub.psi.redhat.com:80"
+)
```
Comment (KPostOffice) [inline advise_reporter.py]: Just use default value option for os.getenv()?
```
os.getenv("PROMETHEUS_PUSHGATEWAY_URL", "pushgateway-dh-prod-monitoring.cloud.datahub.psi.redhat.com:80")
```

File: consumer.py (hunk 3 above)
Comment (fridex) [reply consumer.py]: No argument passed should work just fine.

**Review by KPostOffice — APPROVED** [review]
LGTM :+1:

**Pull request merged**